content/logs/faq/log-collection-troubleshooting-guide.md

```diff
@@ -142,6 +142,32 @@ config_providers:
 
 When using Journald in a containerized environment, make sure to follow the instructions in the [journald integration][4] as there is a specific file used to mount to the Agent.
 
+## Serverless environment
+
+### Logs from lambda functions are not visible in log explorer
+
+See the [Datadog-AWS Log integration][9] to configure your environment.
+
+If you still do not see your logs, double-check the following points:
+
+#### are logs expectedly dropped? 
+
+Check if logs appear in the [Datadog Live Tail][10]. Source is should to be `aws` or `cloudwatch` (see the `parse_event_source()` function in the [Datadog Python lambda function][11] for details).
+
+If they appear in the Live Tail page, you probably explicitly excluded those logs from indexes. 
+
+If they do not appear in the Live Tail page, remind that we drop logs older than 6 hours.
```

Comment:
Maybe we should phrase this differently. Instead of telling them what might be happening, maybe we should ask them to check if they have Exclusion filter on their index or if their logs are in the past.